Documentation/reference/os/mutex.rst

```diff
@@ -0,0 +1,130 @@
+=====================
+Mutual Exclusion lock
+=====================
+
+nxmutex
+=======
+
+Use `nxmutex` prefixed api for protect resources.In fact, nxmutex is implemented
+based on nxsem,The difference between nxmutex and nxsem is that nxmutex supports
+priority inheritance by default, nxsem not support priority inheritance by default
```

Comment:
```suggestion
priority inheritance by default, nxsem do not support priority inheritance by default
```

Reply:
Perhaps it should be `nxsem does not support priority inheritance by default`.

(Also it seems some spaces are missing after periods and commas.)